commands/redis/redis

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#ddev-generated
+## Description: Run redis-cli inside the redis container
+## Usage: redis [flags] [args]
+## Example: "redis KEYS *" or "ddev redis-cli INFO" or "ddev redis-cli --version"
+
+redis-cli -p 6379 -h redis -a redis --no-auth-warning "$@"
```

Comment:
```suggestion
## Description: Run redis-cli inside the redis container
## Usage: redis [flags] [args]
## Example: "ddev redis KEYS '*'" or "ddev redis INFO" or "ddev redis --version"

set -x
redis-cli -p 6379 -h redis -a redis --no-auth-warning "$@"
```

Fixed the command examples.
Quotes around `'*'` - because it is expanded when running from zsh.
Redis commands are so complicated, let's add `set -x` to see what is actually executing.